cob_cartesian_controller/setup.py

```diff
@@ -0,0 +1,13 @@
+#!/usr/bin/env python
```

Comment:
This is for creating a proper python package for the `simple_cartesian_interface`.
See [here](http://docs.ros.org/indigo/api/catkin/html/user_guide/setup_dot_py.html)

BTW, I think you should do the same for your `data_collector` in cob_twist_controller